autoscoper/src/ui/TimelineDockWidget.cpp

```diff
@@ -292,6 +292,46 @@ void TimelineDockWidget::on_spinBox_LastFrame_valueChanged(int d)
   }
 }
 
+void TimelineDockWidget::updateBolding(const unsigned int dof, const bool state)
+{
+  QFont* font;
```

Comment:
```suggestion
  QFont font;
```

and simplify code below accordingly.